recipes/kimi-k2.5/trtllm/agg/nvidia/README.md

```
@@ -23,14 +23,26 @@ This directory contains two aggregated deployment configurations for the `nvidia
 
 ## Standard Aggregated Deployment
 
-Uses [`deploy.yaml`](deploy.yaml). This is the simpler configuration -- aggregated serving with KV-aware routing, no CPU-offloaded KV cache.
+Uses [`deploy.yaml`](deploy.yaml). This is the simpler configuration — aggregated serving with KV-aware routing, no CPU-offloaded KV cache.
+
+Reference configuration for `(kimi-k2.5, B200, ISL/OSL=1024/1024, AGG)`:
+
+| Field          | Value                                       |
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_only; Actual: leading_and_trailing; Unexpected trailing pipe

(MD055, table-pipe-style)